presto-main/src/main/java/io/prestosql/server/testing/TestingPrestoServer.java

```diff
@@ -222,9 +222,8 @@ public TestingPrestoServer(boolean coordinator,
                     binder.bind(TestingStatementAccessControlManager.class).in(Scopes.SINGLETON);
                     binder.bind(AccessControlManager.class).to(TestingAccessControlManager.class).in(Scopes.SINGLETON);
                     binder.bind(EventListenerManager.class).to(TestingEventListenerManager.class).in(Scopes.SINGLETON);
-                    binder.bind(StatementAccessControlManager.class).to(TestingStatementAccessControlManager.class).in(Scopes.SINGLETON);
+                    newOptionalBinder(binder, StatementAccessControlManager.class).setBinding().to(TestingStatementAccessControlManager.class);
```

Comment:
why optional binder?

Reply:
AccessControlModule (which has `newOptionalBinder(binder,StatementAccessControlManager.class);` ) is not added in the existing code of TestingPrestoServer